include/bluetooth/services/hids.h

```diff
@@ -125,11 +125,13 @@ typedef void (*bt_gatt_hids_notif_handler_t) (enum bt_gatt_hids_notif_evt evt);
 
 /** @brief HID Report event handler.
  *
- * @param rep  Pointer to the report descriptor.
- * @param conn Pointer to Connection Object.
+ * @param rep	Pointer to the report descriptor.
+ * @param conn	Pointer to Connection Object.
+ * @param write	true if handler is called for report write.
```

Comment:
*True?

Reply:
```true``` is the value defined in C whereas ```True``` is not. So it becomes slippy here if you stay grammatically correct.

Reply:
used `@c true`